src/Worker/Grpc/InternalOptionsExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.DurableTask.Worker.Grpc.Internal;
+
+/// <summary>
+/// Provides access to configuring internal options for the gRPC worker.
+/// </summary>
+public static class InternalOptionsExtensions
+{
+    /// <summary>
+    /// Configure the worker to use the default settings for connecting to the Azure Managed Durable Task service.
+    /// </summary>
+    /// <param name="options">The gRPC worker options.</param>
+    public static void ConfigureForAzureManaged(this GrpcDurableTaskWorkerOptions options)
```

Comment:
Can this be moved to the `AzureManaged` project? Code in the `Grpc` project is supposed to be backend-agnostic.

Reply:
This is based on a suggestion of mine. The properties being added in `InternalOptions` are specialized for AzureManaged scenario, but they need to be in the project. To avoid having these options that customer's are not expected to ever configure, we make the values themselves `internal` and only expose a single `public` point to configure them.

With that said, we can name this method more backend-agnostic, but it does need to remain in this project so it can access the internal properties. Additionally, we should include a disclaimer in the XML doc explaining this is considered an internal API surface and not subject to breaking change guarantees.

Reply:
I kept the name (we will confuse only ourselves if we invent more names) but there is now an XML doc that states that this is an internal API.